### PR TITLE
Add openj9-systemtest to the build

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Signed-off-by: Mesbah <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

File: systemtest/build.xml [hunk 1]
```diff
@@ -40,6 +40,24 @@
 			<arg value="https://github.com/AdoptOpenJDK/openjdk-systemtest.git" />
 			<arg value="${SYSTEMTEST_ROOT}/openjdk-systemtest" />
 		</exec>
+		<if>
+			<equals arg1="${TARGET}" arg2="openj9" />
```
Comment (sophia-guo) [inline systemtest/build.xml]: What is the TARGET value? I'm confused with this variable.

Comment (Mesbah-Alam) [reply systemtest/build.xml]: The TARGET value is either 'opnej9' or 'system' for sytstemtest. It is what is passed in to maketest.sh in JenkinsfileBase.

File: systemtest/build.xml [hunk 2]
```diff
@@ -40,6 +40,24 @@
...
+	</target>
+	
+	<target name="clone_openj9-systemtest">
+		<exec executable="git" failonerror="false">
+			<arg value="clone" />
+			<arg value="--depth" />
+			<arg value="1" />
+			<arg value="-b" />
+			<arg value="exclude_openj9.test.sharedCLasses.jvmti" />
+			<arg value="https://github.com/Mesbah-Alam/openj9-systemtest.git" />
```
Comment (sophia-guo) [inline systemtest/build.xml]: Is this a temporarily solution using personal repo?

Comment (Mesbah-Alam) [reply systemtest/build.xml]: Yes. Please ignore it for now. I am only using it for testing using my personal builds. I will remove it before delivering.

File: systemtest/playlist.xml [hunk 3]
```diff
@@ -18,12 +18,12 @@
...
 		</subsets>
-		<levels>
-			<level>extended</level>
-		</levels>
 		<groups>
 			<group>system</group>
 		</groups>
+		<levels>
+			<level>extended</level>
+		</levels>
```
Comment (sophia-guo) [inline systemtest/playlist.xml]: This change I don't think is necessary. Same for the following similar changes.

File: systemtest/playlist.xml [hunk 4]
```diff
@@ -121,9 +119,6 @@
 		<levels>
 			<level>extended</level>
 		</levels>
-		<groups>
-			<group>system</group>
-		</groups>
```
Comment (sophia-guo) [inline systemtest/playlist.xml]: Any reason to remove the group system. Will this work?

File: systemtest/exclude_playlist.xml [hunk 5]
```diff
@@ -75,4 +74,33 @@
 			<group>system</group>
 		</groups>
 	</test>
+	
+	<!-- Ref: openj9-systemtest/issues/17 -->
```
Comment (sophia-guo) [inline systemtest/exclude_playlist.xml]: I believe to disable a testcase we could use the new tag <disabled>. No necessary to move the this file.

Comment (smlambert) [reply systemtest/exclude_playlist.xml]: Yes, you should now be able to use the < disabled / > tag within the playlist.xml file

File: systemtest/playlist.xml (hunk 4 above)
Comment (Mesbah-Alam) [reply systemtest/playlist.xml]: It was a mistake. Fixed!

File: systemtest/exclude_playlist.xml (hunk 5 above)
Comment (Mesbah-Alam) [reply systemtest/exclude_playlist.xml]: Update made!

File: systemtest/playlist.xml (hunk 3 above)
Comment (Mesbah-Alam) [reply systemtest/playlist.xml]: Correct. It was a mistake. Fixed!

**Comment by Mesbah-Alam** [comment]
PR tested @ adopt with following personal build: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/282/console

**Comment by Mesbah-Alam** [comment]
Related PR that requires to be delivered prior to delivering this PR: https://github.com/eclipse/openj9-systemtest/pull/18

**Pull request merged**